health/health.d/apcupsd.conf

```diff
@@ -31,7 +33,9 @@ component: UPS
      warn: $this < 100
      crit: $this < (($status == $CRITICAL) ? (60) : (50))
     delay: down 10m multiplier 1.5 max 1h
-     info: average UPS charge over the last minute
+     info: Average UPS charge over the last minute. \
+           UPS battery charge dropped, the UPS is running on battery. It will shut down if external power is not restored. \
+           Prepare IT equipment for the shutdown.
```

Comment:
```suggestion
           The UPS is running on battery power. It will shut down if external power is not restored. \
           You should prepare any attached equipment equipment for the shutdown.
```